docs/backup_and_restore.rst

```diff
@@ -139,6 +139,11 @@ command. Otherwise, you should copy the backup archive that you wish to restore
 Restoring From a Backup File
 ''''''''''''''''''''''''''''
 
+.. important:: This documentation applies to a SecureDrop instance using
+               v2 onion services. If your instance uses v3 onion services,
+               you will need to follow additional steps depending on your
+               specific restore scenario.
+
 To perform a restore, you must already have a backup archive. Provide its
```

Comment:
I would link to `v3_services` here because people who are restoring backups may not know what a v2 vs v3 onion service is, or know how to tell which one their instance has.

Reply:
There will be a separate PR for this, either part of the fix for #4786 or referencing it.

Reply:
:heavy_check_mark: